apps/src/netsim/NetSimPacketEditor.js

```diff
@@ -712,6 +712,9 @@ NetSimPacketEditor.prototype.bindElements_ = function () {
     rowFields.message.keydown(function (jqueryEvent){
       if (isUnmodifiedEnterPress(jqueryEvent)) {
         this.enterKeyPressedCallback_(jqueryEvent);
+        // We don't want to insert a newline or anything, since we have special
+        // handling for the enter key.
+        jqueryEvent.preventDefault();
```

Comment:
It _should_ be unnecessary in this case; there are no other DOM elements with event handlers that it would propagate to here, only the system default behavior (the typing behavior).